sympy/series/limits.py

```diff
@@ -156,17 +169,37 @@ def free_symbols(self):
         return isyms
 
 
-    def doit(self, **hints):
-        """Evaluates limit"""
+    def doit(self, sequence=None, trials=5, deep=True, **hints):
+        """Evaluates the limit.
+
+        Parameters
+        ==========
+
+        sequence : bool or None, optional (default: None)
+            Consider the limit as a sequence limit. If None, the value is
+            inferred from the variable being integer.
```

Comment:
"from the variable having integer assumptions" (being an integer sounds like a literal integer is a valid variable)